charts/camunda-platform/README.md

```diff
@@ -67,6 +68,8 @@ Check out the default [values.yaml](values.yaml) file, which contains the same c
 | | `identity.auth.operate.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Operate. Should be public accessible, the default value works if port-forward to operate is created to 8080. Can be overwritten if, ingress is in use and an external IP is available. | `"http://localhost:8080"` |
 | | `identity.auth.tasklist.existingSecret` |  Can be used to reference an existing secret. If not set, a random secret is generated. The existing secret should contain an `tasklist-secret` field, which will be used as secret for the Identity-Tasklist communication. | ` ` |
 | | `identity.auth.tasklist.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Tasklist. Should be public accessible, the default value works if port-forward to Tasklist is created to 8080. Can be overwritten if, an Ingress is in use and an external IP is available. | `"http://localhost:8080"` |
+| | `identity.auth.optimize.existingSecret` |  Can be used to reference an existing secret. If not set, a random secret is generated. The existing secret should contain an `optimize-secret` field, which will be used as secret for the Identity-Optimize communication. | ` ` |
+| | `identity.auth.optimize.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Tasklist. Should be public accessible, the default value works if port-forward to Tasklist is created to 8080. Can be overwritten if, an Ingress is in use and an external IP is available. | `"http://localhost:8080"` |
```

Comment:
```suggestion
| | `identity.auth.optimize.redirectUrl` |  Defines the redirect URL, which is used by Keycloak to access Optimize. Should be public accessible, the default value works if port-forward to Optimize is created to 8080. Can be overwritten if, an Ingress is in use and an external IP is available. | `"http://localhost:8080"` |
```

Reply:
Though I guess this has to be updated in the values.yaml file, then regenerated.

Reply:
Ok this weird I could swear I have changed this. Thanks for noting this